addons/mail/static/src/messaging/entity/chatter/chatter.js

```diff
@@ -271,6 +283,11 @@ function ChatterFactory({ Entity }) {
         hasThread: attr({
             default: true,
         }),
+        hasScrollAdjust: attr({
+            compute: '_computeHasScrollAdjust',
+            default: true,
+            dependencies: ['hasExternalBorder'],
+        }),
```

Comment:
I may have misunderstand. I know that the prop comes from the form view. But:
- hasExternalBorder is rightly set by the form view since only it knows the chatter's position relative to the form.
- hasScrollAdjust is the responsibility of the chatter (I think) and for now just mirrors hasExternalBorder. We may want to change its computation in the future (including with other props) -- I know future proofing is not an argument, but just an exemplification of the responsibilities.

Reply:
It's the responsibility of the form view not of the chatter, it's the form view that is "allowing space" to the chatter and deciding to put in either in a position where it will share the scroll with the whole page or not.